src/components/DeleteConfirmationPopup.js

```diff
@@ -0,0 +1,32 @@
+import { Popup } from "../components/Popup.js";
+
+export default class DeleteConfirmationPopup extends Popup {
+  constructor({ popupSelector }) {
+    super({ popupSelector });
+    this.popupForm = this._popupElement.querySelector(".modal__form");
+    console.log(this._popupForm);
+    this._modalDeleteButton = this._popupElement.querySelector(
+      ".modal__save-button"
+    );
+  }
+
+  setSubmitAction(action) {
+    this._handleFormSubmit = action;
+  }
+
+  defaultText() {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the check list). It’s common in coding to do it so that functions could `speak` for themselves about what they do.


`set...`